html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML-Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <label for="email">Email</label>
+      <div>
+        <input type="text" name="username" id="email" maxlength="30">
+      </div>
+      <label for="cool">Why is coding cool?</label>
+      <div>
+        <select name="coding" id="cool">
+          <option value="select" selected="selected">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <input type="radio" name="language" value="html" id="HTML">
+      <label for="HTML">HTML</label>
+      <input type="radio" name="language" value="css" id="CSS">
+      <label for="CSS">CSS</label>
+      <input type="radio" name="language" value="javascript" id="JAVASCRIPT">
+      <label for="JAVASCRIPT">Javascript</label>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="sign" value="yes" id="check">
+      <label for="check">Sign me up!</label>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="textcomments" id="comments" cols="30" rows="10"></textarea>
+      <div>
+        <input type="submit" id="finished">
+      </div>
+      <label for="finished"></label>
```

Comment:
you can remove this label, the submit input doesn't need one

Reply:
pushed again with label removed for the submit button